assets/src/infrastructure/TopLevelProviders/index.js

```diff
@@ -0,0 +1,17 @@
+/* Internal dependencies */
```

Comment:
Here we might add some other providers, for example, `ThemeContext` from `styled-components` or similar, if any.